src/Cms/App.php

```diff
@@ -1196,9 +1196,9 @@ public function smartypants(string $text = null): string
      * and return a template snippet
      *
      * @internal
-     * @return string
      * @param mixed $name
      * @param array $data
+     * @return string
```

Comment:
Should be string|null?

Reply:
Yes, that's right. But I think those are errors we need to fix manually when we see them. Not going to read through all doc blocks. ;)

I'm planning to continue with general refactoring for 3.3, so I will fix issues like this over time.